script/check_newsfragments.py

```diff
@@ -82,5 +82,11 @@ def check_file(file: Path, verbose: bool) -> bool:
     return False
 
 
+def log(msg: str) -> None:
```

Comment:
```suggestion
def echo(msg: str) -> None:
```
Nitpick: I would use the same terminology than click, because logging is something else.

Reply:
Makes sense, changed!